DuckDuckGo/AppDelegate.swift

```diff
@@ -28,6 +28,8 @@ import BackgroundTasks
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
+    private static let ShowKeyboardOnLaunchThreshold = TimeInterval(45)
```

Comment:
This is a great quality-of-life improvement! This can be merged once the timing threshold is updated, thanks @st3fan! 🎉 

```suggestion
    private static let ShowKeyboardOnLaunchThreshold = TimeInterval(20)
```